src/perftest/java/com/lmax/disruptor/support/ValueMutationQueueProcessor.java

```diff
@@ -57,7 +58,7 @@ public void halt()
     @Override
     public void run()
     {
-        running = true;
+        running = RUNNING;
         while (true)
```

Comment:
This line of code looks so interested to me. It takes me back to 2014 when I saw my friend create a constant `ZERO = 0`. So, in my point of view, we can keep this `running = true` following `KISS`